javascript-source/core/permissions.js

```diff
@@ -1202,15 +1391,13 @@
     $.isCaster = isCaster;
     $.isAdmin = isAdmin;
     $.isMod = isMod;
-    $.isModv3 = isModv3;
     $.isModeratorCache = isModeratorCache;
     $.isOwner = isOwner;
     $.isSub = isSub;
-    $.isSubv3 = isSubv3;
     $.isTurbo = isTurbo;
     $.isDonator = isDonator;
     $.isVIP = isVIP;
-    $.isReg = isReg;
```

Comment:
I was afraid this will be sth that comes up.
The issue I see (same with `isModv3 `and `isSubv3 `as well as the mappings I left intact) is that the functions do not 100% do the same anymore.
The original `isRegular `checked if the user had _Regular_ permissions or any higher permissions, the new `isReg `function just tell is if the user is exactly that and not if  same as the isSubv3 and isModv3 did at that time.

Checking for permissions like `isRegular` was doing should be done with `$.checkUserPermission(username, tags, permission)` for consistency reasons and to reduce duplicate logic and ensuring the permission check works as intended where used.

I'm hesitant to just map `$.isRegular` to `isReg` because of the above-mentioned possibly breaking custom stuff.
I'm leaning towards just getting the old logic back and marking it as deprecated, leaving the new logic in (which is used throughout the modules).

But I'm open for your take on this

Reply:
In that case we would make a deprecated function which uses the new logic to achieve the same result until a new major release occurs